README.md

```diff
@@ -6,24 +6,19 @@ This is a mobile app project template using [Uno Platform](https://github.com/un
 
 We use `dotnet` project templates to easily create new projects. It simplifies the **project renaming** and supports **conditional inclusions**.
 
-### Installing and uninstalling the template
+### Installing the template
 
-1. In order to install the template, clone this repository on your machine and open a command prompt at the root of the project and run the following command 
+1. In order to install the template, clone this repository on your machine, open a command prompt at its root, and run the following command
 
    `dotnet new -i ./`
 
-
-1. If you want to uninstall the template, run the following command.
-
-    `dotnet new -u ./`
-
     This will list you the list of installed templates, look for this template and copy the command with the absolute path like this. (Note the quotes added, otherwise it doesn't work)
 
     `dotnet new -u "C:\P\ApplicationTemplate"`
 
 ### Running the template to generate a new project
 
-1. To run the template and create a new project, run the following command.
+1. To run the template and create a new project, run the following command in the root folder that will contain the new project.
```

Comment:
run the following command AT the root folder that will contain the new project

Reply:
Verified with Ryan and "in" implies that the user should be inside the folder while running the command